exercises/operations-troubleshooting-00/Setup/setup.sh

```diff
@@ -1,12 +1,25 @@
 #!/usr/bin/env bash
-kubectl exec -n vault-tr-00 vault-tr-00-0 -- vault operator init -key-shares=1 -key-threshold=1 -format=json > ./cluster-keys.json
+
+# reset
+#kubectl exec -it vault-tr-00-0 -n vault-tr-00 -- /bin/sh -c 'rm -rf /vault/data/*'
+#kubectl delete po vault-tr-00-0 -n vault-tr-00
+#rm -rf .terraform* terraform.tfstate* cluster-keys.json ../cluster-keys.json
+
+status=$(kubectl exec -it vault-tr-00-0 -n vault-tr-00 vault status | grep Init | awk '{print $2}')
+
+echo "Vault initialized: $status"
+
+if [[ "$status" != *true* ]]
```

Comment:
I figured it's better to quickly check the status of the Vault. If there is already an initialized Vault, `cluster-keys.json` would get overwritten with repeated `setup.sh`